applications/love/secrets-summit.yaml

```diff
@@ -0,0 +1,60 @@
+auth-ldap-bind-password:
```

Comment:
Only add to this file what is _different_ for the summit. Things that appear everywhere should remain in `secrets.yaml`.